Description: Adds linkage to GLU

# glu-link by Felix Geyer <debfx@ubuntu.com>
Index: b/QGLViewer/QGLViewer.pro
===================================================================
--- a/QGLViewer/QGLViewer.pro	2012-05-26 18:18:44.514662336 +0200
+++ b/QGLViewer/QGLViewer.pro	2012-05-26 18:20:04.476545543 +0200
@@ -147,6 +147,8 @@ unix {
     QMAKE_LIBS_OPENGL *= -lglut
   }
 
+  QMAKE_LIBS_OPENGL *= -lGLU
+
   MOC_DIR = .moc
   OBJECTS_DIR = .obj
 
